src/core/packages/chrome/navigation/src/components/side_nav/panel.tsx

```diff
@@ -40,6 +40,7 @@ export const SideNavPanel = ({ children }: SideNavPanelProps): JSX.Element => {
           ${useEuiOverflowScroll('y')}
           border-right: ${euiTheme.border.width.thin} ${euiTheme.colors.borderBaseSubdued} solid;
           height: 100%;
+          scroll-padding-top: 44px; /* account for fixed header when scrolling to elements */
```

Comment:
@Dosant let's use a CSS variable for this. If we could hook it up to the actual header that'd be even better.

By "fixed header" we mean the "panel fix header", right? We could clarify this in the comment.

Reply:
Thank you 🙇